source/Octopus.Tentacle.Tests.Integration/Support/TentacleConfigurationTestCase.cs

```diff
@@ -47,7 +47,12 @@ public override string ToString()
             string version = Version?.ToString() ?? "Latest";
             builder.Append($"{version},");
             builder.Append($"{SyncOrAsyncHalibut}");
-            builder.Append($",{TentacleRuntime.GetDescription()}");
+
+            var tentacleRuntimeDescription = TentacleRuntime.GetDescription();
+            if (tentacleRuntimeDescription != RuntimeDetection.GetCurrentRuntime())
+            {
+                builder.Append($",{tentacleRuntimeDescription}");
+            }
```

Comment:
It'd be great if we could be more descriptive here e.g. `net6.0client-net48service`, rather than just the framework of the service. The string could be constructed by using the current runtime (there's a helper for this somewhere) and the Tentacle runtime.

Reply:
Good idea.

I nearly went with the suggested style of `net6.0client-net48service`, but remembered how you and Luke didn't like lengthy test names in Rider.

So I went for a more compact version of C and S. If you prefer it to be more verbose, then I'm cool with that. But I thought you might like it shorter 😁 

![image](https://github.com/OctopusDeploy/OctopusTentacle/assets/91929764/b6142d2c-3565-40eb-87df-095b7dc7b2c5)